submissions/krylenger/dom-api-task/script.js

```diff
@@ -0,0 +1,189 @@
+      "They usually feature a single round headlight, simpler rounded clocks, stitched seats, more subtle colour schemes, rounded exhausts, loads of optional shiny bits and often wire spoked wheels or cast wheels designed to look a little bit like spokes from a distance.",
+      "Closely related are modern cafe racers which tend to start life as retro naked bikes but with the addition of low bars, a headlight fairing and pillion seat hump.",
+    ],
+    models: ["Kawasaki Z900RS", "Triumph Speed Twin", "Moto Guzzi V7 III"],
+    avgPrice: "Average price: $10.000",
+  },
+];
+
+const navBtnToggle = document.querySelector(".nav-btn-toggle");
+const nav = document.querySelector(".nav");
+const navUl = document.querySelector(".nav__ul");
+
+const main = document.querySelector(".main");
+const mainHeader = document.querySelector(".main__header");
+const mainImage = document.querySelector(".main__img");
+const mainDescription = document.querySelector(".main__description");
+const mainDescriptionText = document.querySelector(".main__description-text");
+const mainUlBikes = document.querySelector(".main__ul-bikes");
+const mainAvgPrice = document.querySelector(".main__avg-price");
+
+const handleNavBtnToggle = ({ target }) => {
+  if (navBtnToggle.classList.contains("nav-btn-toggle--nav-closed")) {
+    navBtnToggle.classList.remove("nav-btn-toggle--nav-closed");
+    navBtnToggle.classList.add("nav-btn-toggle--nav-opened");
+    nav.classList.add("nav--opened");
+    nav.classList.remove("nav--closed");
+    main.classList.remove("main--wide");
+    main.classList.add("main--short");
+    navUl.classList.remove("nav__ul--display-none");
+    navUl.classList.add("nav__ul--display-yes");
+  } else {
+    navBtnToggle.classList.remove("nav-btn-toggle--nav-opened");
+    navBtnToggle.classList.add("nav-btn-toggle--nav-closed");
+    nav.classList.remove("nav--opened");
+    nav.classList.add("nav--closed");
+    main.classList.remove("main--short");
+    main.classList.add("main--wide");
+    navUl.classList.remove("nav__ul--display-yes");
+    navUl.classList.add("nav__ul--display-none");
+  }
+};
+
+const handleNavigationClick = ({ target }) => {
+  const sectionName = target.textContent;
+  const section = database.find((item) => item.name.toLowerCase() === sectionName.toLowerCase());
+  mainHeader.textContent = section.name;
+  mainImage.setAttribute("src", section.img);
+  mainAvgPrice.textContent = section.avgPrice;
+
+  const fragmentBikes = document.createDocumentFragment();
+  const fragmentDescription = document.createDocumentFragment();
+
+  section.models.forEach((model) => {
+    const bike = document.createElement("li");
+    bike.textContent = model;
+    fragmentBikes.appendChild(bike);
+  });
+  
+  section.description.forEach((description) => {
+    const desc = document.createElement("p");
+    desc.textContent = description;
+    desc.setAttribute("class", "main__description-p");
```

Comment:
I think it's better to use the `classList.add` method for this.

Reply:
Thanks,
✅ Done